client/question/questionSpec.js

```diff
@@ -92,4 +102,19 @@ describe('question.js', function() {
       }
     });
   });
+
+  describe('CORRECTNESS_FEEDBACK_TEXT', function() {
+    it('should match user state constant values with feedback type labels',
+      function() {
+        var correctnessFeedbackTextKeys =
+          Object.keys(CORRECTNESS_FEEDBACK_TEXT);
+        expect(correctnessFeedbackTextKeys).toContain(
```

Comment:
Can you check for exact match to the list of feedback types, rather than just containment?

Reply:
toContain requires an exact match for strings. E.g., toContain would fail for just 'OUTPUT_'. The test checks to make sure the defined feedback type constant values are used in CORRECTNESS_FEEDBACK_TEXT.

Are you suggesting to do the reverse check as well? I.e., for each key in CORRECTNESS_FEEDBACK_TEXT, make sure there is some constant defined with the same value as the key?

Reply:
Yes, that's right (re your second paragraph).

Reply:
Done